### ISSUE TITLE
Feature Ideen Liste

### ISSUE THREAD

**Issue by tsafs** [Hartz-Arbeit/LohntDat#3]
Da wir ja noch gar nicht genau wissen, was wir hier eigentlich entwickeln wollen, dient dieses Issue genau dazu, dies herauszufinden :-)

Ich werfe mal in den Raum, dass ich das Design von "Bruno - Brutto Netto Rechner" ziemlich schick und intuitiv finde und mir so auch für diese App vorstellen kann.

**Comment by pkunze** [comment]
Vom Design abgesehen denke ich MVP wäre alle relevanten Eingaben sammeln, und dann ein Knopf den man drückt und als Ergebnis erstmal nur die Info ob man besser weiter arbeitet oder Harz 4 nimmt. Von dort an könnte man sich dann steigern.

**Comment by IridiumWasTaken** [comment]
Fände es cool, wenn die App sich am Ende einfärbt, um anzugeben, ob sich arbeiten lohnt oder eben nicht (z.B. grüner Hintergrund --> arbeiten lohnt, rot --> arbeiten lohnt nicht). Optionales Feature: Wie lange wird sich arbeiten noch lohnen, wenn die Inflation so hoch bleibt?

**Comment by Sossenbinder** [comment]
Würde noch die Idee in den Raum werfen, dass man eine Art von summary implementiert, die man kompakt und einfach teilen kann, damit man das recht easy an seine Kontakte weiterleiten kann, einfach damit das mehr Leute mitbekommen

**Comment by kevinunger** [comment]
- Finde man sollte nicht auf einen Button klicken müssen bzw. wäre es cool, wenn sich das Ergebnis nach Änderung direkt neu berechnet. So kann man immer fix vergleichen.
- Berechnungsziel umstellen. Man kann einstellen, ob am Ende rauskommt, ob es sich lohnt oder wie viel man verdienen müsste, damit es sich lohnt usw.
- Grafische Darstellungen
- Tooltips zu den Eingabefeldern
- Anzeigen des Defizits. (z.B. mit H4 würdest du 100€ weniger übrig haben)
- Anzeigen ab welchem Verdienst es sich "lohnt"
- Wir sollten die Berechnungsgrundlagen speichern. So könnte man ein Jahr einstellen, wonach berechnet wird. Man könnte dann den Rechner mit älteren Berechnungsgrundlagen nutzen und somit wieder vergleichen. Wäre interessant, wenn eine Reform von H4/Bürgergeld kommt, um vergleichen zu können (inflationsbereinigt), was sich verändert hat.
- Falls sich H4 lohnt, vielleicht erklären, was das bedeutet. Eventuell eine Liste von Sanktionen, Statistiken dazu oder Bedingungen anzeigen. Das es sich finanziell lohnt, ist ja nur die eine Seite der Medaille.

Für den MVP sollten wir uns erstmal auf ein einfaches Formular einigen. Was sind die Eingaben des Nutzers (Nettolohn, Miete, etc.), die wir benötigen, um zu entscheiden, ob sich H4 lohnt? Da müsste sich mal einer melden, der davon Ahnung hat bzw. sich im Detail damit beschäftigt hat :)

**Comment by tsafs** [comment]
Nur für mein Verständnis: Im Prinzip bauen wir einen Brutto/Netto-Rechner, erweitert um konfigurierbare monatliche Fixkosten (z.B. Miete, Essen). Diese werden mit H4 Leistungen verglichen und es wird automatisch ein Go/NoGo ausgerechnet.

Falls dem so wäre, könnte man mit dem Brutto/Netto-Rechner ja schon anfangen oder extern eine API ansprechen, falls es dafür eine API gibt.

**Comment by Tschi02** [comment]
Nein, das Ziel soll es sein, seine aktuellen Einkünfte einzugeben und für Harz-IV relevante Informationen (Immobilienbesitz usw.) und dann gesagt zu bekommen, ob man mit Harz-IV nicht unterm Strich mehr hätte. Ausgangsüberlegung war, dass Harz-IV die in letzter Zeit stark gestiegenen Mietpreise und Heizkosten übernimmt während vor allem Geringverdiener darunter sehr leiden.

**Comment by platypusgit** [comment]
Moin. 
Es gibt natürlich bereits zahlreiche Leute und Gruppen, die sich mit den Berechnungsgrundlagen beschäftigt haben. Ich würde empfehlen auf deren Grundlage zu arbeiten und das Endergebnis dann mit dem Verdienst zu vergleichen. Vor allem ein übersichtlicher Workflow und wäre wichtig, die Berechnungen und Ansprüche sind nämlich um einiges komplizierter und die vorhandenen Lösungen von der UI her... Naja seht selbst.
Nicht sehr hübsch, aber von Harald Thomé, einem sehr bekannten Kämpfer für soziale Politik/bzw. Tacheles e.V., gibt es hier ein Excel-Berechnung für ALG2 auf neuestem Stand. Das gibt schon mal einen kleinen Einblick → https://tacheles-sozialhilfe.de/files/Aktuelles/2022/ALG2-Berechnung-ver-5-0.xlsx

Hier findet ihr eine Übersicht wie SGB2 funktioniert → https://harald-thome.de/files/pdf/2022/SGB%20II%20-%20Folien%2002.08.2022.pdf

**Comment by sebastianrauer** [comment]
Habe ein Figma Jam aufgesetzt. Dort können wir MVP Features voten und weiter Ideen kommunizieren.
https://www.figma.com/file/bQ6ihFro2BHICJwl726G2C/Untitled?node-id=0%3A1
Wenn Figma nicht das richtige ist, gerne andere Tools vorschlagen.
https://imgur.com/a/DnNlDPm

Anschließend würde ich eine Sitemap daraus erstellen und eine grobe Informationsarchitektur machen. Daraus ergibt sich dann das MVP Design.

**Comment by Hartz-Arbeit** [comment]
Weil ich hier Kalkulationen mit Immobesitz lese, bin der Meinung das wir das so einfach wie mögliche halten sollten. Was ist den unsere Zielgruppe? Die die über Hartz tatsächlich nachdenken haben wohl kaum Immobilien.

**Comment by Hartz-Arbeit** [comment]
Meine Vorstellung habe ich hier geschildert: https://github.com/Hartz-Arbeit/LohntDat/issues/1#issuecomment-1208394686

**Comment by kevinunger** [comment]
> Weil ich hier Kalkulationen mit Immobesitz lese, bin der Meinung das wir das so einfach wie mögliche halten sollten. Was ist den unsere Zielgruppe? Die die über Hartz tatsächlich nachdenken haben wohl kaum Immobilien.

Ist völliger Quatsch dort jetzt Immos und Aktien etc einzubauen. Erstmal einen MVP bauen und dann sehen wir weiter.

**Comment by sebastianrauer** [comment]
Hier ein Vorstoß für eine erste Info Archtiecture: https://www.figma.com/file/EXhM4MDGwIKgNF9hqbMwQZ/iA?node-id=4%3A20

**Comment by kevinunger** [comment]
> Hier ein Vorstoß für eine erste Info Archtiecture: https://www.figma.com/file/EXhM4MDGwIKgNF9hqbMwQZ/iA?node-id=4%3A20

![image](https://user-images.githubusercontent.com/5601589/183611806-a83da75e-ba16-4b96-ba0d-70a158be4e8b.png)


:-(

**Comment by sebastianrauer** [comment]
https://www.figma.com/file/EXhM4MDGwIKgNF9hqbMwQZ/iA

**Comment by Julchemie** [comment]
Wie sieht es mit der gesetzlichen Rente aus? Es werden mWn keine Rentenpunkte bei Arbeitslosigkeit gesammelt.

**Comment by platypusgit** [comment]
> Wie sieht es mit der gesetzlichen Rente aus? Es werden mWn keine Rentenpunkte bei Arbeitslosigkeit gesammelt.

Das ist korrekt, es werden keine Rentenversicherungsbeiträge mehr gezahlt, sobald jemand in ALG-II-Bezug ist. Die Zeit gilt allerdings als sogenannte Anrechnungszeit ohne Bewertung, d.h. die "Wartezeit" von 45 Jahren bis zur Altersrente wird angerechnet. Das Jobcenter stellt auch entsprechende Bescheide aus, die bei Überprüfung beim Rentenversicherungsträger für die Zeit der Arbeitslosigkeit vorgelegt werden müssen.

Meiner Meinung nach schießt eine Berücksichtigung von Rentenansprüchen ähnlich wie Immobilienbesitz und Anlagen allerdings etwas über das Ziel der App hinaus.

**Comment by cr4yfish** [comment]
>Meiner Meinung nach schießt eine Berücksichtigung von Rentenansprüchen ähnlich wie Immobilienbesitz und Anlagen allerdings etwas über das Ziel der App hinaus.

Ab in den Backlog damit :D.

**Comment by ByteDream** [comment]
[Discussions](https://github.com/Hartz-Arbeit/LohntDat/discussions) wurden eingerichtet, Features können in der vorhandenen [Features](https://github.com/Hartz-Arbeit/LohntDat/discussions/categories/features) Kategorie vorgeschlagen werden.
Ist handlicher als ein großer Issue Thread

**Comment by cr4yfish** [comment]
Hab gerade gesehen, dass man Issues verlustfrei in eine Discussion umwandeln kann. Ich mach das mal.
